### ISSUE TITLE
Deep Neural Networks with Massive Learned Knowledge

### ISSUE THREAD

**Issue by juanmc2005** [juanmc2005/phd-bib#12]
## [Deep Neural Networks with Massive Learned Knowledge](https://www.semanticscholar.org/paper/Deep-Neural-Networks-with-Massive-Learned-Knowledge-Hu-Yang/e770e1b88351280d810967b3633731c6968ea6d0)
*Regulating deep neural networks (DNNs) with human structured knowledge has shown to be of great benefit for improved accuracy and interpretability. We develop a general framework that enables learning knowledge and its confidence jointly with the DNNs, so that the vast amount of fuzzy knowledge can be incorporated and automatically optimized with little manual efforts. We apply the framework to sentence sentiment analysis, augmenting a DNN with massive linguistic constraints on discourse and polarity structures. Our model substantially enhances the performance using less training data, and shows improved interpretability. The principled framework can also be applied to posterior regularization for regulating other statistical models.*

### Cite this paper
```bibtex
@inproceedings{Hu2016DeepNN,
  title={Deep Neural Networks with Massive Learned Knowledge},
  author={Zhiting Hu and Zichao Yang and Ruslan Salakhutdinov and Eric P. Xing},
  booktitle={EMNLP},
  year={2016}
}
```
